matisse/src/main/java/com/zhihu/matisse/internal/model/AlbumMediaCollection.java

```diff
@@ -51,7 +51,7 @@
         }
 
         return AlbumMediaLoader.newInstance(context, album,
-                album.isAll() && args.getBoolean(ARGS_ENABLE_CAPTURE, false));
+                (AlbumMediaLoader.Capture) args.getSerializable(ARGS_CAPTURE));
```

Comment:
`album.isAll()` condition is also checked in `AlbumMediaLoader.newInstance()` method. In my opinion, this check is not needed here. However, in practice, the capture buttons will only appear in the category "All media".